<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>521</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>bienvenido UwU</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>341</width>
      <height>281</height>
     </rect>
    </property>
    <property name="title">
     <string>bienvenida</string>
    </property>
    <widget class="QPushButton" name="bienvenir">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>100</y>
       <width>87</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>ÒwÓ</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>331</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>introduzca su nombre UwUr</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="name">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>30</y>
       <width>113</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>291</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>introduzca su edad OwO</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="age">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>60</y>
       <width>113</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QListWidget" name="list_bienvenida">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>130</y>
       <width>251</width>
       <height>101</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_ocupacion">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>240</y>
       <width>211</width>
       <height>26</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>ocupaciones disponibles:</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ching cheng chua chuo chichecho </string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>presidente</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>repartidor de pizzas</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>estudiante</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>maestro</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>albañil</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>cientifico</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>vendedor de frutas</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>bombero</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>policia</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>doctor</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>astronauta</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>tortillero</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>pollero</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>nini</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>10</y>
      <width>431</width>
      <height>461</height>
     </rect>
    </property>
    <property name="title">
     <string>calculadora</string>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>66</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>numero 1</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>66</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>numero 2</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="numero1">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>50</y>
       <width>113</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="numero2">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>80</y>
       <width>113</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QListWidget" name="list_sumar">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>240</y>
       <width>256</width>
       <height>211</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_sumar">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>110</y>
       <width>92</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>sumar</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_restar">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>130</y>
       <width>92</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>restar</string>
     </property>
    </widget>
    <widget class="QPushButton" name="calcular">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>200</y>
       <width>87</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>calcular</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkbox_multiplicar">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>150</y>
       <width>92</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>multiplicar</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkbox_dividir">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>170</y>
       <width>92</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>dividir</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>121</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>genero</string>
    </property>
    <widget class="QRadioButton" name="radiomasculinio">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>109</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>masculino</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radiofemeninio">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>109</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>femenino</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radiogay">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>109</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>gay</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>300</y>
      <width>201</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>GroupBox</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
